src/components/sbb-journey-summary/sbb-journey-summary.tsx

```diff
@@ -13,7 +13,7 @@ import { removeTimezoneFromISOTimeString } from '../../global/helpers/timezone-h
 })
 export class SbbJourneySummary implements ComponentInterface {
   /**  The config prop */
-  @Prop() public config!: InterfaceJourneySummaryAttributes['config'];
+  @Prop() public summaryConfig!: InterfaceSbbJourneySummaryAttributes;
```

Comment:
For me it doesn't matter, but why did you change the name from config to summarConfig? In my opinion it is already in context of JourneySummary, so "summary" in the name is redundant.

Reply:
okay, i reverted the naming